### PR TITLE
Feat(Timestamp): Add timestamp to Message Actions (AP-106)

### PR DESCRIPTION
**Pull request by nickjjang**

### CODE REVIEW

File: components/tailored/messaging/message/Message.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
       :setReplyChatbarContent="setReplyChatbarContent"
       :emojiReaction="emojiReaction"
       :editMessage="editMessage"
+      :message="message"
       :messageOwner="this.$props.message.from" />
```
Comment (RetroPronghorn) [inline components/tailored/messaging/message/Message.html:22]: can we nuke the messageOwner prop if we're passing in the whole message?

Comment (nickjjang) [reply components/tailored/messaging/message/Message.html:34]: removed messageOwner

**Review by Jekrimo — APPROVED** [review]

**Comment by nickjjang** [comment]
@RetroPronghorn will you review it again?

File: components/tailored/messaging/message/actions/Actions.vue [hunk 2]
```diff
@@ -37,10 +37,16 @@ export default Vue.extend({
...
+    message: {
+      type: Object as PropType<UIMessage>,
+      default: {
+        id: '0',
+        at: 1620515543000,
+        type: 'text',
+        from: 'group',
+        payload: 'Invalid Message',
+      }
+    }
```
Comment (RetroPronghorn) [inline components/tailored/messaging/message/actions/Actions.vue:49]: comma dangle missing

**Review by RetroPronghorn — CHANGES_REQUESTED** [review]

**Review by RetroPronghorn — APPROVED** [review]

**Pull request merged**